study.md

```diff
@@ -29,21 +29,23 @@ Asynchronous - operations that may start in a certain sequence and may end in an
 In your own words, describe the event loop and how it works.
 
 ```md
-<!-- your answer here -->
+The javascript process that moves functions from the event queue to the staging area into the call stack.
 ```
```

Comment:
Good description of what it is! It looks like you forgot half the question though! Make sure you understand how it's working as well. Find one of us if you need help!